src/modules/compliance/src/lib/Regex.h

```diff
@@ -19,28 +19,40 @@ using regex = std::regex;
 
 #include <memory>
 #include <regex.h>
+#include <regex>
 #include <string>
 namespace RegexLibcWrapper
 {
 class Regex
 {
+    int ConvertFlags(std::regex_constants::syntax_option_type options)
+    {
+        int flags = 0;
+        if (options & std::regex_constants::icase)
+            flags |= REG_ICASE;
+        if (options & std::regex_constants::extended)
+            flags |= REG_EXTENDED;
```

Comment:
nit: wrap in curly parenthesis ({}) the blocks under 'if' even if single line for consistency with rest of project code?